instituto-de-investigaciones-juridicas-unam.csl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-MX">
+  <info>
+    <title>Instituto de Investigaciones Jurídicas UNAM (es-MX)</title>
```

Comment:
Please change this to `Universidad Nacional Autónoma de México - Instituto de Investigaciones Jurídicas (Español)`. You will need to change the self-link, ID and filename accordingly.